browser-test/src/applicant/questions/file.test.ts

```diff
@@ -242,6 +242,128 @@ test.describe('file upload applicant flow', () => {
     })
   })
 
+  test.describe('required file upload question with multiple file uploads', () => {
+    const programName = 'Test program for multiple file upload'
+    const fileUploadQuestionText = 'Required file upload question'
+
+    test.beforeEach(async ({page, adminQuestions, adminPrograms}) => {
```

Comment:
**suggestion, non-blocking**: These can probably be one, maybe two tests with multiple `test.step()`s, instead of individual tests that require a full new database each time.

See https://github.com/civiform/civiform/wiki/Testing#guidelines-for-functional-browser-tests

Reply:
Ah I see. This is a pattern we're trying to move away from cause it's significantly slower but it makes sense to mirror it here.